storybook/main.qml

```diff
@@ -85,9 +105,10 @@ ApplicationWindow {
     }
 
     Settings {
-        id: storeSettings
+        id: settings
 
-        property string selected: ""
-        property bool loadAsynchronously: false
+        property string currentPage: root.currentPage
+        property bool loadAsynchronously: loadAsyncCheckBox.checked
+        property bool nightMode: nightModeCheckBox.checked
```

Comment:
If you use `property alias ...` here, then you don't have to set the `checked` property on the nightMode checkbox :) Same for the other properties

Reply:
I mean, by using an `alias` in settings, it works both ways

Reply:
Cool! I forgot abut that trick, thanks!